ReactViewControl/ReactViewControl.csproj

```diff
@@ -17,7 +18,7 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="WebViewControl-WPF" Version="2.106.1" />
+    <PackageReference Include="WebViewControl-WPF" Version="2.85.23" />
```

Comment:
Then we need to follow up with neves to publish the most corrent version for WebViewControl-WPF